src/components/ButtonSecondary/buttonSecondary.tsx

```diff
@@ -0,0 +1,17 @@
+import React, { PropsWithChildren } from 'react'
```

Comment:
**nitpick:** Component file names should be named with PascalCase, so `ButtonSecondary.tsx` for example